### PR TITLE
SIWE CSS, file and folder naming updates

### PR DESCRIPTION
**Pull request by georgewrmarshall**
### Description

Updating some styles and files to follow BEM css naming conventions. The styles should remain the same.

## Screencast

https://user-images.githubusercontent.com/8112138/168182708-1f31ca7f-f365-4a8b-80c2-6e94976721e6.mov

### CODE REVIEW

File: ui/components/app/signature-request-siwe/README.mdx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
...
 import SignatureRequestSIWE from '.';
 
-# Signature Request
+# Signature Request SIWE
 
-dApp requesting the user to Sign-In with Ethereum.
+dApp requesting the user to Sign in with Ethereum.
 
 <Canvas>
   <Story id="ui-components-app-signature-request-siwe-signature-request-siwe-stories-js--default-story" />
```
Comment (georgewrmarshall) [inline ui/components/app/signature-request-siwe/README.mdx:10]: Story title should match component name

File: ui/components/app/signature-request-siwe/index.scss [hunk 2]
```diff
@@ -1,65 +1,67 @@
-@import 'signature-request-header/index';
```
Comment (georgewrmarshall) [inline ui/components/app/signature-request-siwe/index.scss:1]: Updating styles to use BEM naming convention

File: ui/components/app/signature-request-siwe/index.scss [hunk 3]
```diff
@@ -1,65 +1,67 @@
...
+          user-select: none;
+          -moz-user-select: none;
+          -webkit-user-select: none;
+          cursor: pointer;
+        }
+      }
+    }
+  }
 }
-
```
Comment (georgewrmarshall) [inline ui/components/app/signature-request-siwe/index.scss:65]: Updating styles to use BEM naming convention

**Review by skgbafa — APPROVED** [review]
These changes look good to me. Tested them locally, successfully!

**Pull request merged**